src/acl/external/AD_group/ext_ad_group_acl.cc

```diff
@@ -493,8 +490,9 @@ Recursive_Memberof(IADs * pObj)
         }
         VariantClear(&var);
     } else {
-        if (hr != E_ADS_PROPERTY_NOT_FOUND)
+        if (hr != E_ADS_PROPERTY_NOT_FOUND) {
             debug("Recursive_Memberof: ERROR getting memberof attribute: %s\n", Get_WIN32_ErrorMessage(hr));
+        }
```

Comment:
Instead of adding unwanted curly braces in many places, please fix debug() macro so that it does not require them.

Reply:
They are not unwanted. These braces avoid this compiler error:

```
/home/kinkie/src/squid/mingw/src/acl/external/AD_group/ext_ad_group_acl.cc: In function 'HRESULT Recursive_Memberof(IADs*)':
/home/kinkie/src/squid/mingw/src/acl/external/AD_group/ext_ad_group_acl.cc:498:12: error: suggest explicit braces to avoid ambiguous 'else' [-Werror=dangling-else]
  498 |         if (hr != E_ADS_PROPERTY_NOT_FOUND)
      |            ^
```

Reply:
Please assume that I know about the compiler error you are getting. Think about numerous places where a very similar macro -- debugs() -- is already used without requiring curly braces.

Please fix debug() macro definition so that it doesn't require those braces either (and then undo braces addition without triggering compiler errors you are fixing).

Reply:
I agree with both of you here...

@kinkie is right about these braces being needed. It breaks with Squid coding style - the first portion of the if-statement uses braces, so the `else` should be matching that.

@rousskov is right about the `debug()` macro being broken to produce an error on that badly written code.

Reply:
Now I see what you mean. done in 83203c15e7